universal/components/Button/component.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+import React from 'react'
+
+import styles from './styles.css'
+
+type ButtonProps = {
+  children: any,
+  url: string,
+  segmentProps: Object,
+  disabled: boolean,
+  external: boolean,
+  color: 'dark' | 'light'
+}
+
+function Button ({
+  children,
+  url,
+  color,
+  segmentProps,
+  disabled,
+  external
+}: ButtonProps) {
+  if (disabled) {
+    if (external) {
+      return (
+        <a
+          href={url}
+          className={styles[color] + ' ' + styles.disabled + ' ' + styles.button}
+          target="_blank"
+        >
+          {children}
+        </a>
+      )
+    }
+    return (
+      <a
+        href={url}
+        className={styles[color] + ' ' + styles.disabled + ' ' + styles.button}
+        target="_self"
+      >
+        {children}
+      </a>
+    )
+  }
```

Comment:
this is duplicative. since we're not using react router's link component on this right now, there's only the one case. remove the `external` stuff for now.